staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fq/fq.go

```diff
@@ -0,0 +1,468 @@
+package fq
```

Comment:
in favor of a complete package name instead of acronym?

Reply:
updated package name to `fairqueuing`